### PR TITLE
added .gitignore

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

**Comment by Gr8-Dayne** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: challenges/src/main/java/stacksandqueues/PseudoQueue.java [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
+            forEnqueue.push(forDequeue.pop());
 
+        }
+        forEnqueue.push(value);
     }
 
+    // Dequeue
+    public int dequeue(){
+
+        while(!forDequeue.isEmpty()){
```
Comment (JamesDansie) [inline challenges/src/main/java/stacksandqueues/PseudoQueue.java]: should be;
`        while(!forEnqueue.isEmpty()){
`

File: challenges/src/test/java/stacksandqueues/PseudoQueueTest.java [hunk 2]
```diff
@@ -1,7 +1,39 @@
...
+    @Test public void pseudoQueueDequeueTest() {
+        forEnqueue = new PseudoQueue();
+        forDequeue = new PseudoQueue();
+        forEnqueue.enqueue(1);
+        forEnqueue.enqueue(2);
+        forEnqueue.enqueue(3);
+        System.out.println(forEnqueue.forEnqueue.top);
+        int actual = forEnqueue.dequeue();
+        System.out.println(forEnqueue.forEnqueue.top);
+        assertEquals(1, actual);
```
Comment (JamesDansie) [inline challenges/src/test/java/stacksandqueues/PseudoQueueTest.java]: // Dequeue Test
    @Test public void pseudoQueueDequeueTest() {
        forEnqueue = new PseudoQueue();
        forEnqueue.enqueue(1);
        forEnqueue.enqueue(2);
        forEnqueue.enqueue(3);
        System.out.println(forEnqueue.dequeue());
    }

replace with ^

File: challenges/src/main/java/stacksandqueues/PseudoQueue.java (hunk 1 above)
Comment (Gr8-Dayne) [reply challenges/src/main/java/stacksandqueues/PseudoQueue.java]: Rough night lol.

Thank you.
